libs/wingsdk/src/target-sim/app.ts

```diff
@@ -59,7 +59,8 @@ export class App extends core.App {
   private synthed = false;
 
   constructor(props: core.AppProps) {
-    super(undefined as any, "root");
+    // doesn't allow customize the root id- as used hardcoded in the code
+    super(undefined as any, "root", props);
```

Comment:
In theory I'm not sure why we shouldn't also be able to customize the root id for the `sim` target

Reply:
I agree, we use this string mostly `"root/cloud.TestRunner"`, and in the console too, so we really need to be careful with that...